project/ProjectAutoPlugin.scala

```diff
@@ -75,7 +79,13 @@ object ProjectAutoPlugin extends AutoPlugin {
     // show full stack traces and test case durations
     testOptions in Test += Tests.Argument("-oDF"),
 
-    headerLicense := Some(ALv2("2018", "Dennis Vriend")),
+    headerLicense := Some(
+      HeaderLicense.Custom(
+        """|Copyright (C) 2014 - 2019 Dennis Vriend <https://github.com/dnvriend>
+           |Copyright (C) 2019 - 2019 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
```suggestion
           |Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
```

I don't think we need a range 2019 - 2019. We can add the range 2019 - 2020 in a few days. ;-)